recipes-connectivity/opendds/opendds.inc

```diff
@@ -1,5 +1,6 @@
 SUMMARY = "OpenDDS is an open source C++ implementation of the Object Management Group (OMG) Data Distribution Service (DDS)"
 HOMEPAGE = "https://opendds.org"
+BUGTRACKER = "https://github.com/OpenDDS/OpenDDS/issues"
```

Comment:
Should this be OpenDDS/meta-opendds? -- if users have problems they should probably start here

Reply:
> Hi, It should be the bug tracker of the recipe. So OpenDDS, as it is. https://docs.yoctoproject.org/singleindex.html#term-BUGTRACKER I assume the info the user would like to find about the meta layer itself is in the readme.md file. 

OK, maybe this is a bit confusing for someone not immersed in yocto but the phase "for a recipe" on that site makes it sound like the repo that the recipe itself comes from.

> Br and my best wishes for 2024

Same to you and thanks for your contributions!